samples/react-native/package.json

```diff
@@ -6,7 +6,7 @@
     "postinstall": "patch-package",
     "android": "react-native run-android",
     "ios": "react-native run-ios",
-    "start": "react-native start --experimental-debugger",
```

Comment:
This option has been removed with https://github.com/facebook/react-native/pull/46231